packages/neuron-ui/src/widgets/InputSelect/index.tsx

```diff
@@ -16,14 +18,15 @@ export interface InputSelectProps {
   onChange?: (value: string, arg?: SelectOptions) => void
   value?: string
   placeholder?: string
+  isNewUI?: boolean
 }
 
 function parseValue(value: string, options: SelectOptions[]) {
   const option = options.find(o => o.value === value)
   return option?.value || value
 }
 
-const Select = ({ value, options, placeholder, disabled, onChange, className }: InputSelectProps) => {
+const Select = ({ value, options, placeholder, disabled, onChange, className, isNewUI }: InputSelectProps) => {
   const mounted = useRef(true)
```

Comment:
Why should we add the` isNewUI` ?

Reply:
Cause the `InputSelect` is a base component, and the old one is in use now. So, to avoid rebuilding a new InputSelect component to adjust the new UI, I make a change in the old one.

Reply:
When using `InputSelect` in the new ui, there will exist no old UI. Unless it should be displayed like the old UI.

Reply:
> When using `InputSelect` in the new ui, there will exist no old UI. Unless it should be displayed like the old UI.

Ummm... I see. I'll make double-check and try to replace all of them.